src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+function Calculator() {
+  return (
+    <div className="calculatorContainer">
+      <div className="box display"><span className="displayNumber">0</span></div>
```

Comment:
- [Required] I think it would be better if your `Calculator` component has atleast one child component as stated in the project requirements. I would like to suggest that you factor out the `.display` div into a `Display` component and import the new component into the `Calculator` component. Please see the example code below for details on how you can achieve this.

## Example code
```JavaScript
// Display component

const Display = () => {
  return (
    <div className="box display">
      <span className="displayNumber">0</span>
    </div>
  );
}

export default Display;
```

```JavaScript
// Import the Display component in to Calculator

import Display from "./Display";

function Calculator() {
  return (
    <div className="calculatorContainer">
      <Display />
      <div className="box"><button type="button">AC</button></div>
      <div className="box"><button type="button">+/-</button></div>
      <div className="box"><button type="button">%</button></div>
      <div className="box"><button className="orange" type="button">÷</button></div>
      <div className="box"><button type="button">7</button></div>
      <div className="box "><button type="button">8</button></div>
      <div className="box"><button type="button">9</button></div>
      <div className="box"><button className="orange" type="button">x</button></div>
      <div className="box"><button type="button">4</button></div>
      <div className="box"><button type="button">5</button></div>
      <div className="box"><button type="button">6</button></div>
      <div className="box"><button className="orange" type="button">-</button></div>
      <div className="box"><button type="button">1</button></div>
      <div className="box"><button type="button">2</button></div>
      <div className="box"><button type="button">3</button></div>
      <div className="box "><button type="button" className="orange">+</button></div>
      <div className="box zero"><button type="button" className="thin">0</button></div>
      <div className="box"><button type="button" className="thin">.</button></div>
      <div className="box"><button type="button" className="orange thin">=</button></div>
    </div>
  );
}

export default Calculator;
```

## Project requirements
![Screenshot from 2023-08-18 00-17-37](https://github.com/Graciano1997/math_magicians/assets/16392046/75318e58-4342-4895-9927-16e1e82c40d9)

Reply:
thanks